x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -412,7 +417,35 @@ export const EndpointList = () => {
       }
     >
       {hasSelectedEndpoint && <EndpointDetailsFlyout />}
-      {listData && listData.length > 0 && (
+      {
+        <>
+          <div style={{ display: hasListData ? 'flex' : 'none', maxWidth: 200 }}>
```

Comment:
we want the auto refresh to poll even in the onboarding pages where the table isn't visible

Reply:
can we use `useMemo()` for this object?